src/main/kotlin/com/sourcegraph/cody/agent/CodyAgentServer.kt

```diff
@@ -1,6 +1,33 @@
 package com.sourcegraph.cody.agent
 
-import com.sourcegraph.cody.agent.protocol.*
```

Comment:
In this particular case, the star import might be justified.

Reply:
Discussed offline, let's go back to this tomorrow on standup.